packages/cli/generators/model/index.js

```diff
@@ -215,6 +215,22 @@ module.exports = class ModelGenerator extends ArtifactGenerator {
           Object.assign(this.artifactInfo.modelSettings, {strict: false});
         }
 
+        if (this.artifactInfo.modelSettings) {
+          let output = '';
+          for (var key in this.artifactInfo.modelSettings) {
+            if (output.length > 0) {
+              output += ',\n';
+            }
+            output +=
+              key +
+              ' : ' +
+              JSON.stringify(this.artifactInfo.modelSettings[key], null, 5)
```

Comment:
Why 5 characters for indentation? We use 2 characters everywhere else in LoopBack.

Also what happens when `this.artifactInfo.modelSettings[key]` is an object? I think `JSON.stringify` will return multiple lines of text. Only the first line will preserve indentation, the remaining lines will have too little indent.

```ts
@model({
  custom: {
foo: 'bar'
},
})
```

Let's use Yeoman's [stringify-object](https://www.npmjs.com/package/stringify-object) package please to handle the complex details for us.

```ts
this.artifactInfo.modelSettings = stringifyObject(this.artifactInfo.modelSettings, {
  indent: '  ', // two spaces
  singleQuotes: true,
  inlineCharacterLimit: 80,
});
```

Reply:
Very nice tool, thank you. Commited

Reply:
@yanamg7 I still see `JSON.stringify` being used. Did you perhaps forget to push your changes?